src/components/tests/content-tab-assistant.test.tsx

```diff
@@ -364,6 +421,9 @@ describe( 'ContentTabAssistant', () => {
 
 		( useOffline as jest.Mock ).mockReturnValue( true );
 		rerender( <ContentTabAssistant selectedSite={ runningSite } /> );
+		act( () => {
+			jest.advanceTimersByTime( MIMIC_CONVERSATION_DELAY + 1000 );
+		} );
```

Comment:
I think we don't need to advance timers as in this test case we haven't submitted a prompt.

Reply:
Removed